network/src/network.rs

```diff
@@ -219,7 +219,7 @@ impl NetworkState {
             }
         } else {
             debug!(
-                "Ban session({}) failed: not in peer registry or it is in the whitelist",
+                "Ban session({}) failed: not found in peer registry or on the whitelist",
```

Comment:
`not found in peer registry or it already on the whitelist`?

Reply:
> not found in peer registry or it is on the whitelist

It looks more readable this way.